charts/aws-pca-issuer/values.yaml

```diff
@@ -88,3 +88,9 @@ volumes: []
 
 # Additional VolumeMounts on the operator container.
 volumeMounts: []
+
+# expects input structure similar to https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#poddisruptionbudget-v1-policy EXCEPT pod selector is defined by this helm template's `aws-privateca-issuer.selectorLabels` template function. 
+# for example:
+#   podDisruptinoBudget:
+#     maxUnavailable: 1
```

Comment:
Can you add minAvailable to this example as well? It helps show the user which fields they can use here just to get started. Obviously, they can view the link if they'd like to use other fields

Reply:
I intentionally skipped including minAvailable here.  The k8s link says `maxUnavailable` is "mutually exclusive" of `minAvailable`.  So I think adding both here, even uncommented, suggest to the user they can use both.

Thoughts?

Reply:
added